editor/src/components/editor/shortcuts.spec.browser2.tsx

```diff
@@ -259,6 +259,187 @@ describe('shortcuts', () => {
         canvasRectangle({ x: 186, y: 34, width: 94, height: 55 }),
       )
     })
+
+    describe('inside grid', () => {
+      it('can place element spanning a single cell into the grid and take it out', async () => {
+        const editor = await renderTestEditorWithCode(
+          `import { Scene, Storyboard } from 'utopia-api'
+
+export var storyboard = (
+  <Storyboard data-uid='sb'>
+    <Scene
+      id='playground-scene'
+      commentId='playground-scene'
+      style={{
+        width: 620,
+        height: 548,
+        position: 'absolute',
+        left: 212,
+        top: 103,
+      }}
+      data-label='Playground'
+      data-uid='scene'
+    >
+      <div
+      data-uid='grid'
+        style={{
+          backgroundColor: '#aaaaaa33',
+          width: 527,
+          height: 461,
+          display: 'grid',
+          gridTemplateColumns: '1fr 1fr 1fr 1fr 1fr',
+          gridTemplateRows: '1fr 1fr 1fr 1fr 1fr',
+          contain: 'layout',
+          gridGap: 2,
+          position: 'absolute',
+          left: 46,
+          top: 43,
+        }}
+      >
+        <div
+        data-uid='child'
+        data-testid='child'
+          style={{
+            backgroundColor: '#0162ff',
+            position: 'absolute',
+            left: 13,
+            top: 19,
+            width: 74,
+            height: 59,
+            gridColumn: 3,
```

Comment:
please add a test that converts an absolute child withot gridColumn and gridRow

Reply:
https://github.com/concrete-utopia/utopia/pull/6332/commits/4c5f050ed4e36e790fefeb99e44bc614954f0a16